docs/nginx/events.md

```diff
@@ -0,0 +1,54 @@
+Semantic conventions for Nginx Ingress Controller events
+
+**Status**: [Experimental](../../../document-status.md)
+
+<details>
+<summary>Table of Contents</summary>
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Access](#access)
+- [Error](#error)
+
+<!-- tocstop -->
+
+</details>
+
+This document describes the semantic conventions for events in Nginx Ingress Controller. All Nginx Ingress Controller events MUST use a namespace of `nginx_ingress_controller` in the `event.name` property.
+
+## Access
+
+The event name MUST be `nginx_ingress_controller.access`.
```

Comment:
can we link to the definition of the nginx fields below, e.g. https://nginx.org/en/docs/http/ngx_http_log_module.html#log_format?